src/Appium.Net/Appium/AppiumCommand.cs

```diff
@@ -110,39 +130,50 @@ public class AppiumCommand
                 "/session/{sessionId}/appium/device/pull_file"),
             new AppiumCommand(HttpCommandInfo.PostCommand, AppiumDriverCommand.PullFolder,
                 "/session/{sessionId}/appium/device/pull_folder"),
+            // TODO: Use mobile:getPerformanceData
             new AppiumCommand(HttpCommandInfo.PostCommand, AppiumDriverCommand.GetPerformanceData,
                 "/session/{sessionId}/appium/getPerformanceData"),
+            // TODO: Use mobile:getPerformanceDataTypes
             new AppiumCommand(HttpCommandInfo.PostCommand, AppiumDriverCommand.GetPerformanceDataTypes,
                 "/session/{sessionId}/appium/performanceData/types"),
 
             #region (WinAppDriver) legacy app management
```

Comment:
These are intended for WinAppDriver, so I don't think we should remove them. If they're causing confusion, we could consider moving them to a dedicated class instead.

Reply:
Ah, then, `windows: launchApp` and `windows: closeApp` could be the alternative.
https://github.com/appium/appium-windows-driver?tab=readme-ov-file#windows-launchapp
https://github.com/appium/appium-windows-driver?tab=readme-ov-file#windows-launchapp

Let me update TODO to use those